CHANGELOG.md

```diff
@@ -2,7 +2,69 @@
 
 ## New Features:
 
-No changes to highlight.
+### Theme Sharing 🎨 🤝
+
+You can now share your gradio themes with the world!
+
+After creating a theme, you can upload it to the HuggingFace Hub to let others view it, use it, and build off of it!
+
+### Uploading
+There are two ways to upload a theme, via the theme class instance or the command line.
+
+1. Via the class instance
+
+```python
+my_theme.push_to_hub(repo_name="my_theme",
+                     version="0.2.0",
+                     hf_token="...")
+```
+
+2. Via the command line
+
+First save the theme to disk
+```python
+my_theme.dump(filename="my_theme.json")
+```
+
+Then use the `upload_theme` command:
+
+```bash
+upload_theme\
+"my_theme.json"\
+"my_theme"\
+"0.2.0"\
+"<hf-token>"
+```
+
+The `version` must be a valid [semantic version](https://www.geeksforgeeks.org/introduction-semantic-versioning/) string.
+
+This creates a space on the huggingface hub to host the theme files and show potential users a preview of your theme.
+
+An example theme space is here: https://huggingface.co/spaces/freddyaboulton/dracula_revamped 
+
+### Downloading
+To use a theme from the hub, use the `from_hub` method on the `ThemeClass` and pass it to your app:
+
+```python
+my_theme = gr.Themes.Base.from_hub("freddyaboulton/my_theme")
```

Comment:
```suggestion
my_theme = gr.themes.ThemeClass.from_hub("freddyaboulton/my_theme")
```

Reply:
I think it's better to use `Base` as that has the `set` method defined on it.